specification/README.md

```diff
@@ -170,6 +171,20 @@ The [region](#region) bounding volume specifies bounds using a geographic coordi
 
 Tiles consist of metadata used to determine if a tile is rendered, a reference to the renderable content, and an array of any children tiles.
 
+#### Tile Content
+
+A tile can be associated with renderable content. A tile can either have a single `tile.content` object, or multiple content objects, stored in a `tile.contents` array. The latter allows for flexible tileset structures: for example, a single tile may contain multiple representations of the same geometry data, one as a triangle mesh and one as a point cloud:
+
+![](figures/multiple-contents-geometry.png)
+
+The `content.uri` of each content object refers to the tile's binary content in one of the tile formats that are defined in the [Tile format specifications](#tile-format-specifications)), or another tileset JSON to create a tileset of tilesets (see [External tilesets](#external-tilesets)). 
```

Comment:
> binary content

Not all tile formats are binary any more, notably .gltf

(I see this was copied from some outdated text below)